### DISCUSSION TITLE
Data periodicity

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#41] [General]
In order to forecast the actual population distribution, it is interesting to use data that help us to estimate the seasonal population at a given date.

For this purpose, statistical data such as the amount of waste in a population would help us to estimate whether there is an increase in the population in that area.

However, this type of data is collected over a very long period of time, usually a year, and is not useful for modelling. 

Stakeholders find interesting to have a real-time service, with forecasts created from historical records. To be able to calculate the floating population or population mobility for risk studies.

**Comment by sMorrone** [comment]
Real-time services are for information that must be provided immediately after collection and generally on a continuous basis e.g., to proper monitor and manage wastewater quality for the purposes of sustainable development. Honestly, I don't see real-time services being a suitable solution in this case.  Conversely, it would be useful a seasonal collection of data (e.g., 4 times a year). That would allow forecasts from historical records.

**Comment by AndersOstmanNovogit** [comment]
This issue is currently not included in D4.3.

As I see it, there are two different types of problem, namely
1. Existing data is of insufficient quality. In the case described above, the temporal resolution of population data is too coarse.
2. The requested data sets (real-time or near real time population data) is not available through sensor readings or forecasting models.

I see the forecasting solution to be a research question and not something we can comment on in GO-PEG.
